seatunnel-e2e/seatunnel-connector-v2-e2e/connector-clickhouse-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/ClickhouseSinkCDCChangelogIT.java

```diff
@@ -180,12 +186,15 @@ private void checkSinkTableRows() throws SQLException {
                 Arrays.asList(1L, "A_1", 100),
                 Arrays.asList(3L, "C", 100))
             .collect(Collectors.toSet());
-        Assertions.assertIterableEquals(expected, actual);
+        if (!Arrays.equals(expected.toArray(), expected.toArray())) {
```

Comment:
There seems to be a problem with the parameters here. It should be `if (! Arrays. equals (expected. toArray(), actual)) {`